Tests/CarthageKitTests/DependencySpec.swift

```diff
@@ -84,6 +84,17 @@ class DependencySpec: QuickSpec {
 
 					expect(dependency.name) == "whatisthisurleven"
 				}
+
+                it("should be the directory name if the given URL is relative local path") {
+                    let fileManager = FileManager.default
+                    let previousDirectory = fileManager.currentDirectoryPath
```

Comment:
I'm not sure that `previousDirectory` accurate represents what this is -- would `currentDirectory` make more sense?

Reply:
Looking again I understand why it is named that way a little better -- what about `startingDirectory` instead of `previousDirectory`?

Reply:
Agree. Fixed in https://github.com/Carthage/Carthage/pull/2907/commits/49b3e975c130cadcdde1a49722e9fd35770b5017.